server/kotc_game/src/game.rs

```diff
@@ -253,53 +267,34 @@ impl Game {
                 p.hand.remove(card_index);
                 messages.push(self.message_update_hand(p.hand.clone()));
 
-                Rc::clone(&self.columns).borrow_mut().get_mut(column_index).unwrap().add_card(played_card);
-                messages.push(self.message_update_columns());
-
-                let revealed_card = Rc::clone(&self.columns)
-                    .borrow_mut()
-                    .get_mut(column_index)
-                    .unwrap()
-                    .reveal_previous_card();
-                if let Some(character) = revealed_card {
-                    messages.push(self.message_update_columns());
-
-                    if character == Character::Killer {
```

Comment:
So Killer is now handled somewhere else?

Reply:
Yes, it's been moved.